src/coreclr/src/pal/prebuilt/idl/clrdata_i.cpp

```diff
@@ -1,15 +1,21 @@
-// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
```

Comment:
Shouldn't the license goo be retained?

Reply:
(`git add -p` is your friend!)

Reply:
I like to spell it out `git add --patch` or `git commit --patch`.  It iterates through changes so you can select which to add/commit.

Reply:
I add back the header.

Reply:
also `git add --interactive` or `-i`.

> It iterates through changes

Those changes are called hunks in [libxdiff](https://github.com/git/git/tree/master/xdiff) jargon. :)